docs/source/admin_guide/awss3curl.md

```diff
@@ -0,0 +1,71 @@
+# Using Curl to access AWS S3
+
+In some situations, users may wish to upload content to S3 or download content from an S3 bucket. For example, when attempting [manual backups of QHub's data](./backup.md).
```

Comment:
Do we have standards re line-lenght? I generally suggest having something ~220/180 for md documents and sentence-split to help with PR review.

Reply:
Good idea, and I wonder if this can be imposed automatically. I have added an issue https://github.com/Quansight/qhub/issues/1113 - I haven't taken the time to manually format today.

In general, there may be guidance that you would propose for the entire documentation rather than just this review, so maybe we should make some rules for a top-down review of everything.